src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql

```diff
@@ -260,6 +260,20 @@ VALUES
       ,'PublicSector_Report')
 
+
+INSERT INTO Mtd.SourceConfigForImport
+(SourceDatabaseName,SourceTableName,SourceSchemaName,ColumnNamesToInclude,ColumnNamesToExclude,ColumnNamesToMask,ModelDataToPL,IsQueryBasedImport,SourceQuery,StagingTableName)
+VALUES
+   ('EmployerDemand','CourseDemand','dbo','[Id],[ContactEmailAddress],[OrganisationName],[NumberOfApprentices],[CourseId],[CourseTitle],[CourseLevel],[CourseRoute],[LocationName],[Lat],[Long],[DateCreated],[EmailVerified],[DateEmailVerified],[Stopped],[DateStopped],[ExpiredCourseDemandId],[EntryPoint]','','',0,1
+      ,'select [Id],[ContactEmailAddress],[OrganisationName],[NumberOfApprentices],[CourseId],[CourseTitle],[CourseLevel],[CourseRoute],[LocationName],[Lat],[Long],[DateCreated],[EmailVerified],[DateEmailVerified],[Stopped],[DateStopped],[ExpiredCourseDemandId],[EntryPoint] FROM (SELECT [Id],STUFF([ContactEmailAddress],2,charindex(''''@'''',[ContactEmailAddress])-3,REPLICATE(''''*'''',charindex(''''@'''',[ContactEmailAddress])-3)) As [ContactEmailAddress],[OrganisationName],[NumberOfApprentices],[CourseId],[CourseTitle],[CourseLevel],[CourseRoute],STUFF([LocationName],2,charindex('''' '''',[LocationName])-3,REPLICATE(''''*'''',charindex('''' '''',[LocationName])-3)) As [LocationName],STUFF([Lat],6,charindex(''''.'''',[Lat])+1,REPLICATE(''''*'''',charindex(''''.'''',[Lat])+1)) As [Lat],STUFF([Long],6,charindex(''''.'''',[Long])+1,REPLICATE(''''*'''',charindex(''''.'''',[Long])+1)) As [Long],[DateCreated],[EmailVerified],[DateEmailVerified],[Stopped],[DateStopped],[ExpiredCourseDemandId],[EntryPoint] FROM [dbo].[CourseDemand]) a '
```

Comment:
Do we need to bring PII if it's not used as it's static masked?

Reply:
Instead of avoiding the field completely, at least providing the column as masked. that helps in reporting dashboards

Reply:
How does that help Reporting Dashboards ?

Reply:
This is now changed.